### ISSUE TITLE
Animation problem on second monitor only

### ISSUE THREAD

**Issue by pbosab** [KZDKM/Hyprspace#82]
Hyprland, built from branch main at commit 688fe5c14781c63a1db23d4d02bf239283068ff6  (windowrules: add fullscreenstate field (7466)).
Date: Fri Aug 23 21:42:14 2024
Tag: v0.42.0-62-g688fe5c1, commits: 5131

as in the video, problem exist only on the second monitor.

https://github.com/user-attachments/assets/df677769-1343-4877-8cc1-9be82f67abb6

**Comment by KZDKM** [comment]
Noted, damage might be applied incorrectly to negative monitor offsets. Could you show me you monitor configuration in hyprland.conf?

**Comment by pbosab** [comment]
not having a negative one.
btw, here you go:
```

monitor=HDMI-A-1,2560x1080,0x0,1
monitor=HDMI-A-2,1920x1080,2560x0,1
```

**Comment by brunofavs** [comment]
+1

Also happens on latops

Current monitor configuration
```
################
### MONITORS ###
################
monitor=eDP-1,1920x1080@120,0x0,1.25
monitor=desc:Samsung Electric Company C24F390 HTQJ902227,1920x1080@60,-1920x0,1
```

Only happens on the secondary display, which is the samsung one

**Comment by Aleksandar204** [comment]
Also happens here, reported it in #66 ,monitor config is:
```
monitor=eDP-1,preferred,0x0,1
monitor=desc:Dell Inc. DELL S2419H DPHNDM2, preffered, auto-up, 1
```

Happens only on the DELL

**Comment by myamusashi** [comment]
It's kinda weird when I use hy3 plugin and open 2 windows in 1 group, the problem gone

**Comment by KZDKM** [comment]
> It's kinda weird when I use hy3 plugin and open 2 windows in 1 group, the problem gone

Yeah damage has always been weird as hell on hyprland, still trying to wrap my head around it.

**Comment by KZDKM** [comment]
fixed on the newest commit (hopefully
